dev_swift/00_load_data.ipynb

```diff
@@ -60,16 +60,19 @@
    "metadata": {},
```

Comment:
<p>All names in Swift should use camel case.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>